src/test/resources/pacts/publicapi-connector-get-payment-with-gateway-transaction-id.json

```diff
@@ -0,0 +1,119 @@
+{
+  "consumer": {
+    "name": "publicapi"
+  },
+  "provider": {
+    "name": "connector"
+  },
+  "interactions": [
+    {
+      "description": "get a charge request with authorisation success state",
+      "providerStates": [
+        {
+          "name": "a charge with a gateway transaction id exists",
+          "params": {
```

Comment:
Do we need params here ? There are not being passed to pact test `providerIdIsAvailableWhenPaymentIsSubmitted`

Reply:
yes we need this, they are being passed into https://github.com/alphagov/pay-connector/blob/8da884360e2f5559cf615b367127ae9dc1edafac/src/test/java/uk/gov/pay/connector/pact/TransactionsApiContractTest.java#L61-L67

Reply:
ah, I see!! 👍.